test/features/environment.py

```diff
@@ -139,6 +142,50 @@ def before_scenario(context, scenario):
     # Set personas.
     context.personas = PERSONAS
 
+    for tag in scenario.tags:
```

Comment:
This is cumbersome and redundant when we already have steps to create test datasets and resources.